catfs/mio/compress/algorithm.go

```diff
@@ -9,49 +9,74 @@ import (
 
 var (
 	// ErrBadAlgo is returned on a unsupported/unknown algorithm.
-	ErrBadAlgo = errors.New("Invalid algorithm type")
+	ErrBadAlgo = errors.New("invalid algorithm type")
 )
 
+const (
+	// AlgoUnknown represents an unknown algorithm.
+	// When trying to use it an error will occur.
+	AlgoUnknown = AlgorithmType(iota)
```

Comment:
`AlgoUnknown` is not the same as `AlgoNone`. Shall we bring back `AlgoNone` as well?

Reply:
No, because one wants to use ``AlgoNone``, then the ``compress`` package should not be used. For performance reasons I wanted to avoid stacking readers. Old ``AlgoNone`` e.g. still wrote all the index structure and chunked the data, but just did not compress it. The performance was mild but still there.